### PR TITLE
Add OpenJDK preview information

### PR DESCRIPTION
**Pull request by vyedin**
Needs a review, public links, and a PREVIEW header image.

### CODE REVIEW

File: docs/android/get-started/installation/index.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ This article explains how to setup a physical Android device and
 connect it to a development computer so that the device may be used to
 run and debug Xamarin.Android applications.
 
-## [Preview with Microsoft's OpenJDK Distribution](~/android/get-started/installation/openjdk.md)
```
Comment (vyedin) [inline docs/android/get-started/installation/index.md]: @mgmclemore we can't use "OpenJDK" by itself without violating the trademark. It has to be a Microsoft distribution of OpenJDK

Comment (ghost) [reply docs/android/get-started/installation/index.md]: Can we use **Microsoft OpenJDK Preview**?  We need to try to keep the TOC titles reasonably short.

**Review by ghost — APPROVED** [review]
Looks good to me -- we'll publish when you are ready.

**Comment by vyedin** [comment]
The lawyers asked us to publish another build with an updated TPN, so I'll be making another change.

**Review by ghost — APPROVED** [review]
All good

**Comment by ghost** [comment]
The changes have shown up in `xamarin-docs-pr` but aren't live yet. Perhaps needs @conceptdev final approval?

**Pull request merged**